integration_test/third_party_apps_data/agent/ops-agent/linux/enable_kafka

```diff
@@ -0,0 +1,43 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+metrics:
+  receivers:
+    kafka_metrics:
```

Comment:
It's preferred for the receiver and pipeline IDs to stay the same as the receiver type in the sample configuration, enable_<APP> files and the main `confgenerator/testdata/valid/{LINUX_OR_WINDOWS}/{logging_or_metrics}-receiver_{APP}/input.yaml` test case, because some users look at the `valid` cases for sample configurations as well. Additional more complex test cases could use more complex receiver or pipeline ID. 
E.g. `s/{APP}_metrics/{APP}`, `s/{APP}_pipeline/{APP}`, `s/{APP}_default/{APP}` in the receiver / pipeline IDs

The reason behind that is that Log names are derived from logging receiver IDs, and the dropdown in UI + sample query / dashboard relies on the log name. So the sample dashboards and queries can use that. If users customize it, they can then adjust dashboards and queries accordingly. For consistency, the same convention applies to the sample configurations on the metrics side.

Reply:
Sent a suggestion.